azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/python
 
 variables:
-  genn_stable: '3.3.0'
+  genn_stable: '4_2_0_release'
```

Comment:
this will be merged into master to form the eventual 4.2.0 release. Hum, maybe we need to do the GeNN 4.2.0 release first before we merge/ release this @neworderofjamie ?

Reply:
Ah, but isn't https://github.com/genn-team/genn/archive/4.2.0.zip already the 4.2.0 release? It looks like that on the GeNN website: http://genn-team.github.io/genn/

Reply:
uhm ... @neworderofjamie , do we have a version number confusion (or a Thomas confusion)?

Reply:
So, due to over-running development on my side, master has been a mess and 4_2_0_release has ended up getting fixes back ported to it to. The plan is that there'll be a 4.2.1 release with all the Brian2GeNN fixes to go with the new Brian2GeNN release